AccountService/src/main/java/com/account/config/RandomKeyGenerator.java

```diff
@@ -0,0 +1,21 @@
+package com.account.config;
+
+import java.util.UUID;
+
+
+public class RandomKeyGenerator {
```

Comment:
r u using DB, if yes, maybe we use id like acc #

Reply:
In my db I am saving this random no as Account_No which is the primary_Key of the table